sympy/concrete/summations.py

```diff
@@ -670,6 +673,99 @@ def is_absolutely_convergent(self):
         Sum.is_convergent()
         """
         return Sum(abs(self.function), self.limits).is_convergent()
+    @staticmethod
+    def is_expr_convergent(expr):
+        """
+        Checks for convergence of an expression based on properties of convergence and divergence.
+        Parameters
+        ----------
```

Comment:
This should be `=` 's and the same follows for the code below.

Reply:
I think I'm not sure I understand what you wanted to convey . could you please explain ? I have resolved other suggestions . Thanks for reviewing !

Reply:
Hey @praneethratna I know you might be busy and sorry for bugging you but you could you please explain what exactly I need to change/improve . This being my first PR for sympy, I'm super excited to see this merged  :)

Reply:
I mean about the `--------` be replaced with `=========' which you've done already.